src/Microsoft.DotNet.Tools.Pack/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "dotnet-pack": {
+      "executablePath": "..\\..\\artifacts\\win7-x64\\stage2\\bin\\dotnet-pack.exe"
```

Comment:
@piotrpMSFT or are we talking about the path? I think if I say dotnet.exe and set args to pack it won't work as it may attach to the wrong process although I have not tested it